pkg/bootstrapper/bootstrapper.go

```diff
@@ -37,13 +36,15 @@ const (
 	// KubeletServiceName is the name will we run the kubelet Windows service under. It is required to be named "kubelet":
 	// https://github.com/kubernetes/kubernetes/blob/v1.16.0/cmd/kubelet/app/init_windows.go#L26
 	KubeletServiceName = "kubelet"
+	// kubeletDependentSvc is the name of the service dependent on kubelet Windows service
+	kubeletDependentSvc = "hybrid-overlay-node"
 	// kubeletSystemdName is the name of the systemd service that the kubelet runs under,
 	// this is used to parse the kubelet args
 	kubeletSystemdName = "kubelet.service"
 	// kubeletPauseContainerImage is the location of the image we will use for the kubelet pause container
 	kubeletPauseContainerImage = "mcr.microsoft.com/oss/kubernetes/pause:1.3.0"
 	// serviceWaitTime is an arbitrary amount of time to wait for the Windows service API to complete requests
-	serviceWaitTime = time.Second * 10
+	serviceWaitTime = time.Second * 20
```

Comment:
timeout required now to stop two services and refresh.

Reply:
Please update the comment to reflect that.

Reply:
done